Content.Server/StationEvents/Components/RampingStationEventSchedulerComponent.cs

```diff
@@ -14,4 +16,10 @@ public sealed partial class RampingStationEventSchedulerComponent : Component
 
     [DataField("timeUntilNextEvent"), ViewVariables(VVAccess.ReadWrite)]
     public float TimeUntilNextEvent;
+
+    /// <summary>
+    /// The gamerules that the scheduler can choose from
+    /// </summary>
+    [DataField(required: true)]
+    public List<EntitySpawnEntry> ScheduledGameRules = new();
 }
```

Comment:
Aren't these supposed to be components on the entity anyway.

Reply:
Some use type to subset the events like the secret gamerule, but that doesn't allow for using them in multiple different places with different balance if you wanted to. I didn't remove the type schema though because I didn't want to mess up the secret game rule or something else I hadn't considered.